products/views.py

```diff
@@ -0,0 +1,72 @@
+import random
+
+from django.views import View
+from django.http import JsonResponse
+
+from . import models
```

Comment:
```suggestion
from .models import (
    Product,
    ProductDetailImage
)
```

models에서 사용할 클래스를 하나씩 import 하시는 것을 추천드립니다.